drivers/flash/flash_ll_nrfxlib.c

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <errno.h>
+
+#include <zephyr.h>
+#include <device.h>
+#include <soc.h>
+#include <flash.h>
+#include <misc/util.h>
+
+#include "ble_controller_soc.h"
+#include "multithreading_lock.h"
+
+#include <logging/log.h>
+#define LOG_MODULE_NAME blectrl_flash
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
LOG_LEVEL missing?

Reply:
Looks like it, but all logging statements has been removed, so this can be removed as well :)

- [ ] remove unused logging declarations